configuration/collector.yaml

```diff
@@ -116,13 +116,26 @@ providers:
             - name: GremlinRequestsPerSec
               statistic: Sum
             - name: SparqlRequestsPerSec
-              statistic: Sum              
+              statistic: Sum
           period: 24h 
           start_time: 168h # 24h * 7d
           constraint:
             formula: GremlinRequestsPerSec + SparqlRequestsPerSec
             operator: "=="
             value: 0
+      kinesis:
+        - description: Kinesis Put Record(s) Sum (Bytes)
+          metrics:
+            - name: "PutRecords.Bytes"
+              statistic: Sum
+            - name: "PutRecord.Bytes"
+              statistic: Sum              
+          period: 24h 
+          start_time: 168h # 24h * 7d
+          constraint:
+            formula: "[PutRecord.Bytes] + [PutRecords.Bytes]"
```

Comment:
why did you add `[`? for support `.` char?

Reply:
Added documentation.